sources/packages/web/src/views/student/NoticeOfAssessment.vue

```diff
@@ -64,6 +70,10 @@ export default defineComponent({
       type: Number,
       required: true,
     },
+    currentAssessmentId: {
+      type: Number,
+      required: false,
+    },
```

Comment:
Can the `currentAssessmentId` be provided from the event `assessmentDataLoaded` that is already exposing `applicationStatus` and `noaApprovalStatus`?

Reply:
Updated